src/main/java/com/code_intelligence/jazzer/driver/FuzzTargetRunner.java

```diff
@@ -367,6 +367,14 @@ public static int startLibFuzzer(List<String> args) {
       // https://github.com/llvm/llvm-project/blob/da3623de2411dd931913eb510e94fe846c929c24/compiler-rt/lib/fuzzer/FuzzerFlags.def#L19
       args.add("-len_control=100");
     }
+
+    for (String arg : args.subList(1, args.size())) {
+      if (arg.length() > 0 && arg.charAt(0) != '-') {
+        System.out.print("INFO: seed corpus: input file: ");
```

Comment:
The arguments may be directories, not just "input files". This also usually includes the generated corpus (the first non-dash arg), which is not generally called "seed corpus". Is this output format something we could still change?

Reply:
Sure, we can still change that! Do you have a proposal for the format?

Reply:
Maybe `"using input(s) from: " + arg`?

That avoids fuzzing terminology as well as having to determine whether this points to a file or a directory.

@bertschneider What do you think?

Reply:
Fine by me :+1:

Reply:
Done! I removed the parentheses for easier parsing.